testing/integration/linux_deb_test.go

```diff
@@ -272,6 +295,12 @@ func testDebUpgrade(t *testing.T, upgradeFromVersion *version.ParsedSemVer, info
 	out, err := cmd.CombinedOutput() // #nosec G204 -- Need to pass in name of package
 	require.NoError(t, err, string(out))
 
+	newRunDir, err := atesting.FindRunDir(endFixture)
+	require.NoError(t, err, "failed at getting run dir")
+	require.NotEqual(t, runDir, newRunDir, "the run dirs from upgrade should not match")
+	newRunMigrationMarket := filepath.Join(newRunDir, migrationMarkerFile)
+	require.FileExistsf(t, newRunMigrationMarket, "%q is missing", newRunMigrationMarket)
```

Comment:
I guess you were hungry at this point 😉. 

```suggestion
	require.NotEqual(t, runDir, newRunDir, "the run dirs from upgrade should not match")
	newRunMigrationMarker := filepath.Join(newRunDir, migrationMarkerFile)
	require.FileExistsf(t, newRunMigrationMarker, "%q is missing", newRunMigrationMarker)
```

Reply:
I did add a comment about the hash here 0318eabbaac05abd71cf4be3a9c795d93d8047b3. Feel free to unresolve if you spot anything else 🙂